src/Actions/Player/Friends.cs

```diff
@@ -38,29 +38,29 @@ public class Friends : GameAction
         public override void Execute(ActionInput actionInput)
         {
             IController sender = actionInput.Controller;
-            if ((actionInput.Params.Count() == 1 && actionInput.Params[0].ToLower() == "list") || actionInput.Params.Count() == 0)
+            if (actionInput.Params.Length == 1 && actionInput.Params[0].ToLower() == "list" || !actionInput.Params.Any())
             {
                 if (playerBehavior.Friends.Count == 0)
                 {
                     sender.Write("You currently have no friends listed.");
                 }
                 else
                 {
-                    StringBuilder sb = new StringBuilder();
+                    var sb = new AnsiBuilder();
```

Comment:
nit: Should we find all ` sb = new AnsiBuilder` and refactor-rename them to `ab`?  (We can follow up with such cleanup separately too.)

Reply:
Absolutely